README.rst

```diff
@@ -105,3 +105,17 @@ The example code assumes you've set the following environment variables:
 
     # Send all data and stop the exporter
     newrelic.stop()
+
+
+Find and use your data
+------------
```

Comment:
This line results in syntax errors:

```
Checking dist/opencensus_ext_newrelic-0.1.4.dev13+g46e5fe2-py2.py3-none-any.whl: FAILED
  `long_description` has syntax errors in markup and would not be rendered on PyPI.
    line 111: Warning: Title underline too short.

    Find and use your data
    ------------
Checking dist/opencensus-ext-newrelic-0.1.4.dev13+g46e5fe2.tar.gz: FAILED
  `long_description` has syntax errors in markup and would not be rendered on PyPI.
    line 111: Warning: Title underline too short.

    Find and use your data
    ------------
```

```suggestion
----------------------
```

Reply:
Okay I think I signed the CLA and it looks like you have put in an edit to make the header underline thing longer (or if you haven't , could you do that for me? just struggling with trying to figure out all these repos I'm in right now). Let me know if you need anything else from me.